test/data/trusted-html.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset=utf-8 />
+	<title>body</title>
+</head>
+<body>
+<div id="qunit-fixture"></div>
+<script src="../../dist/jquery.min.js"></script>
+<script src="iframeTest.js"></script>
+<script>
+	var i, input, elem, tags,
+		results = [],
+		policy = trustedTypes.createPolicy( "jquery-test-policy", {
+			createHTML: function( html ) {
+				return html;
+			}
+		} ),
+		inputs = [
+			[ "<div></div>", "<div class='test'></div>", [ "div" ] ],
```

Comment:
Does it make sense to also add a test for `TrustedHTML`s that wrap over a non-obvious HTML, e.g. just some text? From what I can tell, different branches are used then.

Reply:
Yes, good point. That will only apply to the second array items as the first ones are passed directly to `jQuery` which also supports selectors - and since for TrustedHTML we cannot carve out a part of an input string to later parse, I opted to restrict support for TrustedHTML-wrapped strings to ones that are start with `<` and end with `>` - i.e. what was already going through the fast path skipping the regex match.

Reply:
So, I assume that `TrustedHTML` wrapping `something` will just get stringified somewhere and be treated as a selector? That's OK, I don't think there would be expectations for jQuery to do anything else. I think it just makes sense to have an explicit test for that behavior.

Reply:
To be clear, this limitation only applies to (pseudo-code) `jQuery(TrustedHTML('foo'))`; `jQuery(document.body).append(TrustedHTML('foo'))` should append the text `foo` just as without wrapping in TrustedHTML; I'll add a test for that.

Reply:
Oh, I forgot about https://api.jquery.com/jQuery/#jQuery-object behavior :) Yeah, that's fine, thanks for clarifying.

Reply:
Sounds good!